### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by cloudprogramm**

### CODE REVIEW

**Review by alexandra-halytska — APPROVED** [review]

File: tests/test_restore_names.py [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+        },
+        {
+            "last_name": "Adams",
+            "full_name": "Mike Adams",
+        },
+    ]
+    restore_names(users)
+
+    for data in users:
+        assert data["first_name"] in data["full_name"]
```
Comment (MasterpieceElbow) [inline tests/test_restore_names.py:54]: It would be better to assert that users should equal to expected value, your implementation is implicit. Add test where you pass correct data to the `restore_names`.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

File: tests/test_restore_names.py [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+        {
+            "last_name": "Adams",
+            "full_name": "Mike Adams",
+        },
+    ]
+
+    assert restore_names(users) is None
+
+
+def test_is_function_set_correct_first_name():
```
Comment (IvanRamyk) [inline tests/test_restore_names.py:20]:
```suggestion
def test_function_should_set_correct_first_name():
```
It's better to specify how the function should behave in the name of the test

File: tests/test_restore_names.py [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+    users = [
+        {
```
Comment (IvanRamyk) [inline tests/test_restore_names.py:22]: Please, add separate tests for both cases (`None` and missing key).
In that case, it'd be easier to find where a bug has occurred if the function does something wrong

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

**Review by IvanRamyk — APPROVED** [review]
